ViewModel/ExpressCheckoutButton.php

```diff
@@ -113,4 +123,20 @@ public function getFormSelector(?string $formSelector, ?ProductInterface $produc
 
         return $formSelector;
     }
+
+    /**
+     * @return array
+     */
+    public function getPostcodeOptionalCountries(): array
+    {
+        $optionalCountries = [];
+        $getOptionalCountries = $this->storeConfig->getValue(
+            'general/country/optional_zip_countries',
+            ScopeInterface::SCOPE_STORE
+        );
+        if ($getOptionalCountries) {
+            $optionalCountries = explode(',', $getOptionalCountries);
+        }
+        return $optionalCountries;
+    }
```

Comment:
is this function still needed?

Reply:
Yes we use it to get the getOptionalCountries from the store settings

Reply:
No, i will remove it